lib/src/ops/levels/calLevSec.js

```diff
@@ -18,26 +13,25 @@ OUT:
 
 // UNCOMMENT FOR INITIAL TESTING
 // calSecLevels()
-/** Main function. */
-export async function calSecLevels() {
-  const inputIfcFilename = getArgInputFilename(getUsage())
-  const elevValuesAll = await extractHeight(inputIfcFilename)
+/**
+ * Main function.
+ * @param {object} model
+ * @param {object} config
+ * @return {array}
+ */
+export async function calSecLevels(model, config) {
+  const elevValuesAll = await extractHeight(model)
```

Comment:
Looks great. I also happened to make similar changes on my end today to have ifc model as input:

https://github.com/mkeshavarzi/ifctool/commit/96f7e7259a15136ac71155b171f8766f128d67ff

I